ingest/rules/curate.smk

```diff
@@ -78,6 +97,7 @@ rule curate:
                 --abbr-authors-field {params.abbr_authors_field:q} \
             | augur curate apply-geolocation-rules \
                 --geolocation-rules {input.geolocation_rules:q} \
+            | python {input.urls_script:q} \
             | augur curate apply-record-annotations \
                 --annotations {input.annotations:q} \
                 --id-field {params.annotations_id:q} \
```

Comment:
`augur curate apply-record-annotations` no longer works as intended – it's using `--id-field PPX_accession` but [ingest/defaults/annotations.tsv](https://github.com/nextstrain/mpox/blob/f9c1b689e145a6ffb4ffa3111b2d0f028e466582/ingest/defaults/annotations.tsv) still uses INSDC accessions.